DNN Platform/Library/Entities/Modules/Settings/SettingsRepository.cs

```diff
@@ -63,23 +63,16 @@ public void SaveSettings(ModuleInfo moduleContext, T settings)
 
                         if(string.IsNullOrEmpty(settingValueAsString))
                         {
-                            if (settingValue is DateTime)
-                            {
-                                settingValueAsString = ((DateTime)settingValue).ToString("u");
-                            }
-                            else
-                            {
-                                settingValueAsString = settingValue.ToString();
-                            }
+                            settingValueAsString = GetSettingValueAsString(settingValue);
```

Comment:
But when I'm specifying `InvariantCulture` then it doesn't matter what locale the site is executing under, unless I'm missing something.